test-harness.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
```

Comment:
This is just a general observation, but sandbox errors seem to be swallowed somewhere in the output. 

```
➜  py-algorand-sdk git:(trim-indexer) 

... (truncated)

Starting sandbox for: harness
see sandbox.log for detailed progress, or use -v.
* docker containers started!
* waiting for services to initialize.
* services ready!

algod version
make: *** [harness] Error 1

➜  py-algorand-sdk git:(trim-indexer) 
```

Sandbox logs does have the error though - would there be a way to pass a `-v` or something to echo this back into the terminal? If not, it's not a huge deal for the user to parse through the sandbox logs either.

Reply:
I explored this a bit, but passing verbose seemed to break the build (however, there were some other unrelated conditions which were different simultaneously, so I may be overly pessimistic). I commit to exploring this some more. 

Also, @algochoi :  I'm wondering what your error was. It seems like that is something worth fixing.

Reply:
Thanks for looking into it - I'm happy with resolving as is. The error was a docker-related issue - I don't have the exact logs anymore but docker failed due to some container name so I killed all my existing containers

Reply:
Maybe the run didn't succeed to bring down the pre-existing images? That's something it's supposed to be able to handle.

Reply:
I can't seem to replicate the error, but I think it could have been an error on Docker's side. The Docker UI was malfunctioning and required some permissions to delete containers so I had to remove it on the terminal manually.

Reply:
leaving open as i want to investigate -v

Reply:
This is now working in verbose mode